src/VerifyGitHubVulnerabilities/Verify/PackageVulnerabilitiesVerifier.cs

```diff
@@ -113,15 +149,150 @@ public Task UpdateVulnerabilityAsync(PackageVulnerability vulnerability, bool wi
                     if (versionRange.Satisfies(version) != package.VulnerablePackageRanges.Contains(existingRange))
                     {
                         Console.Error.WriteLine(
-                            $@"Vulnerability advisory {vulnerability.GitHubDatabaseKey
+                            $@"[Database] Vulnerability advisory {vulnerability.GitHubDatabaseKey
                             }, range {range.PackageId} {range.PackageVersionRange}, package {package.NormalizedVersion
                             } is not properly marked vulnerable to vulnerability!");
                         HasErrors = true;
                     }
                 }
             }
+        }
 
-            return Task.CompletedTask;
+        private Task VerifyVulnerabilityInMetadata(PackageVulnerability gitHubAdvisory)
```

Comment:
Nit: please add an `Async` suffix on methods that should be `await`ed.

```suggestion
        private Task VerifyVulnerabilityInMetadataAsync(PackageVulnerability gitHubAdvisory)
```

Reply:
Some of the other changes resulted in this method no longer being awaitable, but point taken--thanks.